README.md

```diff
@@ -1 +1,17 @@
-# opencost-grafana-dashboard
\ No newline at end of file
+# opencost-grafana-dashboard
+
+Collection of OpenCost Grafana dashboards.
+
+## Dashboards
+### alerting
+Currently no dashboards available. Contributions are welcome.
+
+### cost-reporter
+- opencost-cost-reporter-basic-overview-v0.0.1
```

Comment:
Do we want to have version numbers in the name? That doesn't seem useful, since we'll have to update in 2 places and I don't think it really matters.

Reply:
I thought it could be useful to know if something got changed (new fix, new feature, etc.). But for now I agree to remove the version. I just removed it.